<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugMemoryWindow</class>
 <widget class="QWidget" name="DebugMemoryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mémoire</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>191</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Navigation</string>
   </property>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
    <property name="displayIntegerBase">
     <number>16</number>
    </property>
   </widget>
   <widget class="QPushButton" name="displayButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>29</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Afficher</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>ROM BANK 0 [0x0000]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ROM BANK 01..NN [0x4000]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>VRAM [0x8000]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RAM Externe [0xA000]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>WRAM0 [0xC000]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>WRAM1 [0xD000]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Copie WRAM0 [0xE000]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>OAM [0xFE00]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>JOYP [0xFF00]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DIV [0xFF04]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TIMA [0xFF05]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TMA [0xFF06]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TAC [0xFF07]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>IF [0xFF0F]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LCDC [0xFF40]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SCY [0xFF42]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SCX [0xFF43]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LY [0xFF44]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BGP [0xFF47]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>WY [0xFF4A]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>WX [0xFF4B]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>HRAM [0xFF80]</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>IE [0xFFFF]</string>
     </property>
    </item>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>201</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>Mémoire</string>
   </property>
   <widget class="QTableWidget" name="tableMemoryWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>131</width>
      <height>262</height>
     </rect>
    </property>
    <property name="autoScrollMargin">
     <number>16</number>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>100</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>26</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>23</number>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>0x0</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>0x1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>0x2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>0x3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>0x4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>0x5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>0x6</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>0x7</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>0x8</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>0x9</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Mémoire</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="downButton">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>50</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="upButton">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>30</y>
      <width>23</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>359</x>
     <y>299</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Fermer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refreshButton">
   <property name="geometry">
    <rect>
     <x>263</x>
     <y>299</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Rafraîchir</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>DebugMemoryWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refreshButton</sender>
   <signal>clicked()</signal>
   <receiver>DebugMemoryWindow</receiver>
   <slot>refreshDebugMemoryWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>upButton</sender>
   <signal>clicked()</signal>
   <receiver>DebugMemoryWindow</receiver>
   <slot>refreshUpDebugMemoryWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>13</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>downButton</sender>
   <signal>clicked()</signal>
   <receiver>DebugMemoryWindow</receiver>
   <slot>refreshDownDebugMemoryWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>11</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>displayButton</sender>
   <signal>clicked()</signal>
   <receiver>DebugMemoryWindow</receiver>
   <slot>displayAtSelectedMemoryAddress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>458</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>DebugMemoryWindow</receiver>
   <slot>displayAtDefinedMemoryAddress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>refreshDebugMemoryWindow()</slot>
  <slot>refreshUpDebugMemoryWindow()</slot>
  <slot>refreshDownDebugMemoryWindow()</slot>
  <slot>displayAtSelectedMemoryAddress()</slot>
  <slot>displayAtDefinedMemoryAddress()</slot>
 </slots>
</ui>
